arviz/plots/kdeplot.py

```diff
@@ -331,12 +331,22 @@ def _fast_kde(x, cumulative=False, bw=4.5, xmin=None, xmax=None):
     return density, xmin, xmax
 
 
-@conditional_jit
+@conditional_jit(cache=True)
 def _histogram(x, n_bins, range_hist=None):
     grid, _ = np.histogram(x, bins=n_bins, range=range_hist)
     return grid
 
 
+@conditional_jit(cache=True)
+def cov_(data):
+    return np.cov(data)
+
+
+@conditional_jit(cache=True)
+def stack(x, y):
```

Comment:
should be `def stack(tple):`, I think, or change wherever it is called to use 2 inputs.

![image](https://user-images.githubusercontent.com/2295568/61152788-f1d45500-a4ae-11e9-876c-a861cffdaa22.png)